test/hotspot/jtreg/compiler/lib/ir_framework/CompilePhase.java

```diff
@@ -81,6 +102,10 @@ public enum CompilePhase {
     MATCHING("After matching", RegexType.MACH),
     MACH_ANALYSIS("After mach analysis", RegexType.MACH),
```

Comment:
Could you move this line to after `POSTALLOC_EXPAND("Post-Allocation Expand", RegexType.MACH)`, for consistency with the order in which phases are defined in `phasetype.hpp`?

Reply:
Yes, fixed now. In general, I suspect there are other inconsistencies between `CompilePhase.java` and `phasetype.hpp`. I'll add it as an item to the cleanup RFE.